app/uploaders/facility_image_uploader.rb

```diff
@@ -0,0 +1,35 @@
+class FacilityImageUploader < CarrierWave::Uploader::Base
+
+  include CarrierWave::RMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
```

Comment:
Yes, I will use ENV variable as mentioned above. I'm not merging this yet. I will fix some of your comments. Hopefully, by tomorrow I will have more info about ENV settings.